### PR TITLE
[Component with its own class name as service type ] LPS-196226 Refactor Registry classes from multiple modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-196226

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistry.java [hunk 1]
```diff
@@ -126,31 +122,28 @@ protected void activate(
...
 	protected void deactivate() {
-		_serviceTracker.close();
+		_serviceTrackerMap.close();
 
-		_serviceTracker = null;
+		_serviceTrackerMap = null;
 	}
 
 	private BundleContext _bundleContext;
 	private final Map<ServiceReference<ServletContext>, JSConfig> _jsConfigs =
```
Comment (shuyangzhou) [inline modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistry.java]: @MrJohn1911 this looks really odd, can you just make this "_jsConfigs" the service tracker map and avoid adding the EagerServiceTrackerCustomizer?
The service tracking logic does not seem to have any side affects, there should be no reason to be eager.

**Comment by shuyangzhou** [comment]
@MrJohn1911 @rafaprax see above comment, please resend.

**Pull request closed without merging**